app.json

```diff
@@ -211,12 +211,57 @@
           "template": "add_devices"
         }
       ]
+    },
+    {
+      "id": "tempest",
+      "name": {
+        "en": "Tempest",
+        "nl": "Tempest",
+        "de": "Tempest"
+      },
+      "class": "sensor",
+      "capabilities": [
+        "measure_temperature",
+        "measure_humidity",
+        "measure_pressure",
+        "measure_luminance",
+        "measure_uv",
+        "measure_wind_strength",
+        "measure_wind_angle",
+        "measure_gust_strength",
+        "measure_lightningstrike_count",
+        "measure_lightningstrike_distance",
+        "measure_rain",
+        "measure_rain_day",
+        "measure_solarradiation",
+        "measure_voltage"
+      ],
+      "images": {
+        "large": "/drivers/air/assets/images/large.png",
+        "small": "/drivers/air/assets/images/small.png"
+      },
+      "pair": [
+        {
+          "id": "list_devices",
+          "template": "list_devices",
+          "navigation": {
+            "next": "add_devices"
+          },
+          "options": {
+            "singular": false
+          }
+        },
+        {
+          "id": "add_devices",
+          "template": "add_devices"
+        }
+      ]
     }
   ],
   "flow": {
     "triggers": [
       {
-        "id": "rain_start",
```

Comment:
Doesn't renaming the ids for flow triggers break existing flows users have configured?

Reply:
I am new to Homey development but that makes sense. Let keep them the id the same.